src/containers/CampaignList/components/CampaignListRow.tsx

```diff
@@ -53,126 +46,78 @@ export const CampaignListRow: React.FC<Props> = (props) => {
     externalSystem
   } = campaign;
 
-  let listItemStyle = {};
-  let leftIcon;
-  if (isArchived) {
-    listItemStyle = inlineStyles.past;
-  } else if (!isStarted || hasUnassignedContacts) {
-    listItemStyle = {
-      color: theme.palette.warning.dark
-    };
-    leftIcon = <WarningIcon />;
-  } else if (hasUnsentInitialMessages) {
-    listItemStyle = {
-      color: theme.palette.info.dark
-    };
-  } else {
-    listItemStyle = {
-      color: theme.palette.success.dark
-    };
-  }
   const dueBy = DateTime.fromISO(campaign.dueBy || "");
   const creatorName = campaign.creator ? campaign.creator.displayName : null;
-  let tags = [];
-  if (DateTime.local() >= dueBy) {
-    tags.push({
-      title: "Overdue",
-      color: theme.palette.grey[900],
-      backgroundColor: theme.palette.error.main
-    });
-  }
-
-  if (externalSystem) {
-    const title = `${externalSystem.type}: ${externalSystem.name}`;
-    tags.push({ title, backgroundColor: blue[300] });
-  }
-
-  if (!isStarted) {
-    tags.push({ title: "Not started" });
-  }
-
-  if (hasUnassignedContacts) {
-    tags.push({ title: "Unassigned contacts" });
-  }
-
-  if (isStarted && hasUnsentInitialMessages) {
-    tags.push({ title: "Unsent initial messages" });
-  }
-
-  if (isStarted && hasUnhandledMessages) {
-    tags.push({ title: "Unhandled replies" });
-  }
 
-  if (isStarted && !isArchived && isAutoassignEnabled) {
-    tags.push({ title: "Autoassign eligible" });
-  }
-
-  tags = tags.concat(teams.map(({ title }) => ({ title })));
-  if (campaignGroups) {
-    tags = tags.concat(
-      campaignGroups.edges.map(({ node }) => ({ title: node.name }))
-    );
-  }
+  const headerTags = makeCampaignHeaderTags({
+    isStarted,
+    hasUnassignedContacts,
+    hasUnsentInitialMessages,
+    hasUnhandledMessages
+  });
 
-  const primaryText = (
-    <div style={inlineStyles.chipWrapper}>
-      {campaign.title}
-      {tags.map((tag) => (
-        <Chip
-          key={tag.title}
-          label={tag.title}
-          style={{
-            ...inlineStyles.chip,
-            color: tag.color,
-            backgroundColor: tag.backgroundColor
-          }}
-        />
-      ))}
-    </div>
-  );
-  const secondaryText = (
-    <span style={inlineStyles.secondaryText}>
-      <span>
-        Campaign ID: {campaign.id}
-        <br />
-        {campaign.description}
-        {creatorName ? <span> &mdash; Created by {creatorName}</span> : null}
-        <br />
-        {dueBy.isValid ? dueBy.toFormat("DD") : "No due date set"}
-      </span>
-    </span>
+  const isCampaignSelected = campaignDetailsForExport.find(
```

Comment:
Nit: I like to reserve the is prefix for a real boolean
```suggestion
  const campaignSelected = campaignDetailsForExport.find(
```